qiita_db/analysis.py

```diff
@@ -200,39 +198,39 @@ def delete(cls, _id):
         QiitaDBUnknownIDError
             If the analysis id doesn't exist
         """
-        with TRN:
+        with qdb.sql_connection.TRN:
```

Comment:
You're probably correct, but I did a search/replace. Do you think it is needed to change or it is ok as it is?